terraform/modules/daos_server/main.tf

```diff
@@ -47,20 +47,18 @@ locals {
       access_points = local.access_points
     }
   )
-  server_startup_script = file(
-  "${path.module}/templates/daos_startup_script.tftpl")
-
-  configure_daos_content = templatefile(
```

Comment:
You may still need a configure_daos (without the dmg format and the container creation) as it was intended to be run on clients without the daos image (i.e. slurm, how do you start the agents now that you can read the YML files).